_data/paths/accessible.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
```

Comment:
Did you take a look at putting this data inside of `content/en/accessible/accessible.json` to create a [directory data file](https://www.11ty.io/docs/data-template-dir/) instead? I think it would then get inherited into each page's data object along with the page's YAML.

This might make it easier because then you wouldn't need to look up the collection that a particular page is in when you're generating the next page link. You would know that `topics` is always the topics specific to `accessible`.

Reply:
Yes, I tried that first actually. The problem is that directory data file if visible only to templates that are children of that directory. If we want a guide to belong to 2 paths / collections in future, it might be more beneficial to keep this data independent of actual directory structure. This was the reason why I decided to put it in global data, just in separately named files.

wdyt?

Reply:
Yep that makes sense. I think I'm leaning toward saying that a guide should only be able to live inside of one collection. I'm mostly thinking from a user experience perspective it might be strange to see the same guides popping up over and over but in different places.

Reply:
Even if we limit ourselves to one collection, "upgrading" a post to a collection guide would be easier if it's not tied to directory structure.

Reply:
Ah yeah that's a good point. Ok let's keep the _data approach.